### PR TITLE
Write index release note page for v6.9

### PR DESCRIPTION
**Pull request by robertapplin**
### Description of work
This PR adds the highlights of v6.9 to the release note index page. It also updates the image - in this case it is a small collage of two of the new algorithms added to Mantid.

### To test:
Check the index page for any mistakes. Make sure the image is rendered as expected.

### Reviewer

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

#### Code Review

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Do the release notes conform to the [release notes guide](https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html)?

#### Functional Tests

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### Gatekeeper

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.

### CODE REVIEW

File: docs/source/release/v6.9.0/index.rst [hunk 1]
```diff
@@ -16,7 +16,14 @@ Mantid 6.9.0 Release Notes
```
Comment (backmari) [inline docs/source/release/v6.9.0/index.rst:16]: Should we remove the section above this line? The one about:
```
.. warning:: This release is still under construction. ...
```

**Review by backmari — COMMENTED** [review]
Other than the one comment about the warning, the index page looks good to me! The links are working, except for two that I believe will be generated on the release date: the DOI for the release and the GitHub release page.

File: docs/source/release/v6.9.0/index.rst (hunk 1 above)
Comment (robertapplin) [reply docs/source/release/v6.9.0/index.rst:16]: Oops I missed that, thanks!

**Review by backmari — APPROVED** [review]

**Comment by sf1919** [comment]
DOI and Github release links will be sorted out at release as you correctly summise @backmari :) 

I'm happy so I will get this merged in.

**Pull request merged**